tests/systemd_testsuite/test_01_basic.pm

```diff
@@ -18,7 +18,36 @@ use utils 'zypper_call';
 use power_action_utils 'power_action';
 
 sub run {
-    select_console 'root-console';
+    #prepare test
+    assert_script_run 'cd /var/opt/systemd-tests';
+    assert_script_run './run-tests.sh TEST-01-BASIC --setup 2>&1 | tee /tmp/testsuite.log', 600;
+    assert_script_run 'ls -l /etc/systemd/system/testsuite.service';
+    #reboot
+    power_action('reboot', textmode => 1);
+    assert_screen('linux-login', 600);
+    type_string "root\n";
+    wait_still_screen 3;
+    type_password;
+    wait_still_screen 3;
+    send_key 'ret';
```

Comment:
You should be able to replace

```perl
    assert_screen('linux-login', 600);
￼    type_string "root\n";
￼    wait_still_screen 3;
￼    type_password;
￼    wait_still_screen 3;
￼    send_key 'ret';
```

to

```perl
    select_console('root-console');
```

in each test

Reply:
Yeah, I was trying that before, but I ran into timeouts that cause the test to fail.
It seems that select_console times out after 30s and I didn't find a way to extend the timeout.
Is there one? If so I'm happy to replace this.